packages/pds/src/api/app/bsky/actor/createScene.ts

```diff
@@ -0,0 +1,233 @@
+import { Server, APP_BSKY_SYSTEM, APP_BSKY_GRAPH } from '../../../../lexicon'
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { PlcClient } from '@atproto/plc'
+import * as crypto from '@atproto/crypto'
+import * as handleLib from '@atproto/handle'
+import * as locals from '../../../../locals'
+import * as schema from '../../../../lexicon/schemas'
+import { RepoStructure } from '@atproto/repo'
+import { TID } from '@atproto/common'
+import { UserAlreadyExistsError } from '../../../../db'
+import * as repoUtil from '../../../../util/repo'
+
+export default function (server: Server) {
+  server.app.bsky.actor.createScene(async (_params, input, req, res) => {
+    const { db, auth, config, keypair, logger } = locals.get(res)
+    const { recoveryKey } = input.body
+
+    const requester = auth.getUserDid(req)
+    if (!requester) {
+      throw new AuthRequiredError()
+    }
+
+    const handle = input.body.handle.toLowerCase()
```

Comment:
hmmm right. it's out here because it needs to be normalized before running through validation logic & we need to return the normalized version

It could be normalized in `ensureValid` but that's kinda overloading it & I feel like it'd be too easy to drop the normalized version. 

Maybe a new method `normalizeAndEnsureValid` 🤔

Reply:
At the end of the day it's no biggie :) Main goal in my mind is to couple things closely enough to ensure no uppers make their way into the db, since it's an assumption we make about how the handles are stored.  One easy solution to that which wouldn't involve moving things around is to add an assertion inside `preregisterDid()`.